cli/src/main/scala/com/gu/pfi/cli/Main.scala

```diff
@@ -17,14 +17,14 @@ import scala.concurrent.duration.Duration
 object Main extends App with Logging {
   import scala.language.reflectiveCalls
 
-  val options = new Options(args)
+  val options = new Options(args.toIndexedSeq)
 
   options.subcommand match {
     case Some(options.hashCmd) =>
       HashFiles.run(options.hashCmd.files())
 
-    case Some(cmd @ options.loginCmd) =>
```

Comment:
I have no idea why this stopped working but it's arguably clear afterwards anyway

Reply:
maybe worth us testing the CLI before merging, though if it type-checks it's probably fine